crates/bitwarden-wasm-internal/src/pure_crypto.rs

```diff
@@ -46,6 +46,41 @@ impl PureCrypto {
             .encrypt_with_key(&SymmetricCryptoKey::try_from(key)?)?
             .to_buffer()
     }
+
+    pub fn decrypt_userkey_with_masterpassword(
```

Comment:
We usually write the user key as `user_key`, we should try to use that here too for consistency

Reply:
And `master_password`.

Reply:
Updated.